packages/core/src/util/decorators.ts

```diff
@@ -152,6 +156,12 @@ export function makePropDecorator(
       const decoratorInstance = new (<any>PropDecoratorFactory)(...args);
 
       function PropDecorator(target: any, name: string) {
+        // target is undefined with standard decorators. This case is not supported and will throw
+        // if this decorator is used in JIT mode with standard decorators.
+        if (target === undefined) {
+          throw new RuntimeError(RuntimeErrorCode.UNSUPPORTED_STANDARD_DECORATORS, ngDevMode && 'Standard decorators are not supported in JIT mode.');
```

Comment:
I think there might be a risk of executing this code before we had a chance to init the dev mode global. I think a safer option would be to also use a `typeof`.
```suggestion
          throw new RuntimeError(RuntimeErrorCode.UNSUPPORTED_STANDARD_DECORATORS, (typeof ngDevMode === 'undefined' || ngDevMode) && 'Standard decorators are not supported in JIT mode.');
```